fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/translator/PhysicalPlanTranslator.java

```diff
@@ -138,8 +147,13 @@ public PlanFragment visitPhysicalOlapScan(
         List<Slot> slotList = olapScan.getOutput();
         PhysicalOlapScan physicalOlapScan = olapScan.getOperator();
         OlapTable olapTable = physicalOlapScan.getTable();
+        List<Expr> execConjunctsList = physicalOlapScan
+                .getExpressions()
+                .stream()
+                .map(e -> ExpressionTranslator.translate(e, context)).collect(Collectors.toList());
         TupleDescriptor tupleDescriptor = generateTupleDesc(slotList, context, olapTable);
         OlapScanNode olapScanNode = new OlapScanNode(context.nextNodeId(), tupleDescriptor, olapTable.getName());
+        olapScanNode.addConjuncts(execConjunctsList);
         context.addScanNode(olapScanNode);
```

Comment:
Vectorized and non-vectorized code use different properties, such as `vconjunct` and `conjuncts`.
Our principle is that the properties used by both engines need to be translated before the vectorized code is stable.
After the vectorized code runs stably, the useless code can be deleted

Reply:
converted at NereidPlanner